src/app/containers/PageHandlers/withVariant/getVariantRedirectUrl/index.js

```diff
@@ -0,0 +1,46 @@
+import pathToRegexp from 'path-to-regexp';
+import {
+  getVariant,
+  variantSanitiser,
+  variants,
+} from '#lib/utilities/variantHandler';
+import { getPreferredVariant } from '#app/contexts/UserContext/cookies';
+
+const getVariantRedirectUrl = ({ path, params }, service, variant) => {
+  const cookieVariant = getPreferredVariant(service);
+  const defaultVariant = getVariant({
+    service,
+    variant: variantSanitiser(variant),
+  });
+
+  if (!path) return null;
```

Comment:
would this be better at the top of the function?

Reply:
Fixed!